### PR TITLE
[16.0][mig] stock_partner_delivery_window

### PR DESCRIPTION
**Pull request by aktiv-heli-kantawala**

### CODE REVIEW

File: stock_partner_delivery_window/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                    utc_start = start_time
+                    utc_end = end_time
+                if utc_start <= date_time.time() <= utc_end:
+                    return True
+        return False
+
+    def _get_delivery_time_format_string(self):
+        return _("From %s to %s")
+
+    def get_delivery_time_description(self):
```
Comment (nguyenminhchien) [inline stock_partner_delivery_window/models/res_partner.py:104]: @heliaktiv find no place call this method (`get_delivery_time_description`)

**Review by nguyenminhchien — CHANGES_REQUESTED** [review]
- Functional test: ok
- pre-commit check: need to make it pass

**Comment by nguyenminhchien** [comment]
Because there's no feedback from the author for a long time, i created a new PR https://github.com/OCA/stock-logistics-workflow/pull/1378 to replace this one.

**Comment by rousseldenis** [comment]
@heliaktiv Do you agree if we switch to #1378 ?

**Comment by simahawk** [comment]
@heliaktiv ping

**Comment by simahawk** [comment]
Enough inactivity here and we've got a completed work with reviews on #1378

**Pull request closed without merging**